### PR TITLE
DOP-2596: Only add top margin in connected steps

### PR DESCRIPTION
**Pull request by i80and**
### Stories/Links:

DOP-2596

### Current Behavior:

* **Connected with unconnected**: [https://docs-mongodbcom-integration.corp.mongodb.com/master/guides/heli/master/atlas/account/#create-your-mongodb-account.](https://docs-mongodbcom-integration.corp.mongodb.com/master/guides/heli/master/atlas/account/#create-your-mongodb-account.)
* **Unconnected**: [https://docs-mongodbcom-integration.corp.mongodb.com/master/guides/heli/master/atlas/network-connections/#select-my-local-environment.](https://docs-mongodbcom-integration.corp.mongodb.com/master/guides/heli/master/atlas/network-connections/#select-my-local-environment.)
* **Connected**: [https://docs-mongodbcom-integration.corp.mongodb.com/master/docs/heli/DOP-2596-original/#import-your-data](https://docs-mongodbcom-integration.corp.mongodb.com/master/docs/heli/DOP-2596-original/#import-your-data)

### Staging Links:

* **Connected within unconnected**: [https://docs-mongodbcom-integration.corp.mongodb.com/master/guides/heli/DOP-2596/atlas/account/#create-your-mongodb-account.](https://docs-mongodbcom-integration.corp.mongodb.com/master/guides/heli/DOP-2596/atlas/account/#create-your-mongodb-account.)
* **Unconnected**: [https://docs-mongodbcom-integration.corp.mongodb.com/master/guides/heli/DOP-2596/atlas/network-connections/#select-my-local-environment.](https://docs-mongodbcom-integration.corp.mongodb.com/master/guides/heli/DOP-2596/atlas/network-connections/#select-my-local-environment.)
* **Connected**: [https://docs-mongodbcom-integration.corp.mongodb.com/master/docs/heli/DOP-2596/#import-your-data](https://docs-mongodbcom-integration.corp.mongodb.com/master/docs/heli/DOP-2596/#import-your-data)

### CODE REVIEW

File: src/components/Procedure/Step.js [hunk 1]
```diff
@@ -24,6 +24,16 @@ const StyledStep = styled('div')`
   }
 `;
 
+const StyledUnconnectedStep = styled('div')`
```
Comment (seungpark) [inline src/components/Procedure/Step.js:27]: minor nitpick. we can the headings' stylings with the `landingStepStyles` (seems like there are two places we are using `stepStyle` to determine styling)

**Review by seungpark — APPROVED** [review]
staged changes look great 👍 thank you for the fix! minor nonblocking comment below.

EDIT: and failing snapshots need to be updated via `npm run test -- -u` to run jest with -u flag

File: src/components/Procedure/Step.js (hunk 1 above)
Comment (i80and) [reply src/components/Procedure/Step.js]: ooh, good call -- thank you!

**Comment by i80and** [comment]
@seungpark It looks like `npm run test -- -u` adds my changes to the reference files, but there's a LOT of unrelated stuff -- moving lines around, etc. Would you like me to add the whole shebang, or just patch add the changes relevant to this PR?

**Comment by seungpark** [comment]
> @seungpark It looks like `npm run test -- -u` adds my changes to the reference files, but there's a LOT of unrelated stuff -- moving lines around, etc. Would you like me to add the whole shebang, or just patch add the changes relevant to this PR?

i think the output of the updated snapshots as is, is fine. i see that the snapshots failing are related:

● renders steps nested in include nodes
  ● renders with "connected" styling by default
etc

**Review by seungpark — APPROVED** [review]
changes LGTM 👍  thank you!

**Pull request merged**